class/defaults.yml

```
@@ -2,3 +2,45 @@ parameters:
   csi_exoscale:
     =_metadata: {}
     namespace: syn-csi-exoscale
+
+    manifestVersion: v0.29.6
+    kustomizeInput:
+      namespace: ${csi_exoscale:namespace}
+
+    images:
+      csi_driver:
+        registry: docker.io
+        repository: exoscale/csi-driver
+        tag: 0.29.6
+      csi_attacher:
+        registry: registry.k8s.io
+        repository: sig-storage/csi-attacher
+        tag: v4.4.1
+      csi_resizer:
+        registry: registry.k8s.io
+        repository: sig-storage/csi-resizer
+        tag: v1.9.0
+      csi_provisioner:
+        registry: registry.k8s.io
+        repository: sig-storage/csi-provisioner
+        tag: v3.6.0
+      csi_snapshotter:
+        registry: registry.k8s.io
+        repository: sig-storage/csi-snapshotter
+        tag: v6.3.0
+      snapshot_controller:
+        registry: registry.k8s.io
+        repository: sig-storage/snapshot-controller
+        tag: v6.3.0
+      csi_registrar:
+        registry: registry.k8s.io
+        repository: sig-storage/csi-node-driver-registrar
+        tag: v2.9.0
+      liveness_probe:
+        registry: registry.k8s.io
+        repository: sig-storage/livenessprobe
+        tag: v2.11.0
+
+    apiSecret:
+      accessKey: ?{vaultkv:${cluster:tenant}/${cluster:name}/exoscale/csi_driver/s3_access_key}
+      secretKey: ?{vaultkv:${cluster:tenant}/${cluster:name}/exoscale/csi_driver/s3_secret_key}
```

I've updated the install instructions to not use `s3_` prefix for the Vault secret for the CSI driver IAM credentials, since that makes very little sense for the CSI driver.

```suggestion
      accessKey: ?{vaultkv:${cluster:tenant}/${cluster:name}/exoscale/csi_driver/access_key}
      secretKey: ?{vaultkv:${cluster:tenant}/${cluster:name}/exoscale/csi_driver/secret_key}
```